pyocd/commands/repl.py

```diff
@@ -99,7 +99,3 @@ def run_one_command(self, line):
                 traceback.print_exc()
         except exceptions.CommandError as e:
             print("Error:", e)
-        except Exception as e:
-            print("Error:", e)
-            if session.Session.get_current().log_tracebacks:
-                traceback.print_exc()
```

Comment:
Instead of removing this exception handler, we probably need to add a handler for `ToolExitException`. Otherwise exceptions like `KeyError` or `IndexError` caused by invalid arguments to a command would cause the REPL to exit.